### PR TITLE
Shankhanil branch

### PR DESCRIPTION
**Pull request by Shankhanil**
Introduced SocialBot : a bot framework that automates tweets

### CODE REVIEW

File: src/Quotes.py [hunk 1]
```diff
@@ -7,10 +7,27 @@ class Quotes:
     def __init__(self):
         pass
     
-    def getQuote(self):
-        url = "https://quote-garden.herokuapp.com/api/v2/quotes/random"
+    def generateQuote(self, author = 'random'):
+        parsed = {}
```
Comment (chhandasaha) [inline src/Quotes.py:11]: What is the use of "parsed" variable ?

File: src/SocialBot.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+# Upcoming bot support : Whatsapp
+
+
+import tweepy
+import os
+
+class SocialBot:
+    def __init__(self, platform = 'twitter'):
+        self.use_environment_variables = False
+        self.credential_list = ["CONSUMER_KEY", "CONSUMER_SECRET", "ACCESS_KEY", "ACCESS_SECRET"]
```
Comment (chhandasaha) [inline src/SocialBot.py:14]: why is it needed ?

File: src/Quotes.py (hunk 1 above)
Comment (Shankhanil) [reply src/Quotes.py:11]: the "parsed" variable is a dictionary variable. The REST API sends the quote in the form of JSON object, which is stored in a dictionary variable for easy access.

File: src/SocialBot.py (hunk 2 above)
Comment (Shankhanil) [reply src/SocialBot.py:14]: Credential list is a list of credential variables that is used to log into twitter account. 
We are using Twitter API for automatically tweeting an users favorite quote. To do that, we need to first login into the user's twitter account, and for that, we need a list of login credentials, which are CONSUMER_KEY, CONSUMER_SECRET, ACCESS_KEY and ACCESS_SECRET

**Pull request merged**